.changeset/great-actors-work.md

```diff
@@ -0,0 +1,20 @@
+---
+'@udecode/plate-core': minor
+'@udecode/plate-link': major
+---
+
+`@udecode/plate-core`:
+
+- Add `sanitizeUrl` util to check if URL has an allowed scheme
```

Comment:
@12joan can you create one changeset for each package? We do so when the changelog is not related